src/EffectID.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
-enum class EffectID : Int32
+// tolua_begin
+
+enum EffectID
```

Comment:
That doesn't seem to work. I'm pretty green with tolua but is this relevant?
http://www8.cs.umu.se/kurser/TDBD12/VT04/lab/lua/tolua++.html
>New on tolua++: All other preprocessor directives are ignored.